nutstash-wallet/umbrel-app.yml

```
@@ -0,0 +1,31 @@
+manifestVersion: 1
+id: nutstash-wallet
+category: Finance
+name: Nutstash Wallet
+version: "0.1.6"
+tagline: Nutstash is a Cashu wallet for your browser.
+description: >-
+  This app is in early development and most likely buggy. It does handle real sats though, so be sure to use only amounts that are small enough to lose.
+  
+  What is Cashu?
+
+  "Cashu is a free and open-source Chaumian ecash system built for Bitcoin. Cashu offers near-perfect privacy for users of custodial Bitcoin applications. Nobody needs to knows who you are, how much funds you have, and with whom you transact with." - cashu.space
+  
+releaseNotes: >-
+  - Initial release on the umbrel app store
+  
+developer: Gandlafbtc
+website: https://nutstash.app/
+dependencies: 
```

This needs to be an array.

```suggestion
dependencies: []
```